deployments/polygon/usdc/configuration.json

```diff
@@ -34,31 +43,13 @@
       "supplyCap": "500000e18"
     },
     "WBTC": {
-      "address": "0xc907E116054Ad103354f2D350FD2514433D57F6f",
-      "priceFeed": "0xDE31F8bFBD8c84b5360CFACCa3539B938dd78ae6",
+      "address": "0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6",
+      "priceFeed": "0xc907E116054Ad103354f2D350FD2514433D57F6f",
```

Comment:
This is the `BTC / USD` price feed, although Chainlink also provides a `WBTC / USD` price feed:

https://polygonscan.com/address/0xc907E116054Ad103354f2D350FD2514433D57F6f#readContract

Reply:
I think we should use the `WBTC` price feed since it's available.

Reply:
sounds good. update to:

https://polygonscan.com/address/0xDE31F8bFBD8c84b5360CFACCa3539B938dd78ae6#readContract